### PR TITLE
docs：Add logo of registered enterprise,and Change image source to Alicdn

### PR DESCRIPTION
**Pull request by lvekee**
### Ⅰ. Describe what this PR did
Add logo of registered enterprise,and Change image source to Alicdn

### CODE REVIEW

**Review by lightClouds917 — APPROVED** [review]
LGTM

**Review by pengten — APPROVED** [review]

File: changes/en-us/1.5.0.md [hunk 1]
```diff
@@ -44,7 +44,7 @@
   - [[#4293](https://github.com/seata/seata/pull/4293)] get global lock in the file mode
   - [[#4335](https://github.com/seata/seata/pull/4335)] Realize configuration center upload configuration interactive script (nacos,etcd3)
   - [[#4332](https://github.com/seata/seata/pull/4332)] Realize configuration center upload configuration interactive script (apollo,consul,zk)
-  
+
```
Comment (pengten) [inline changes/en-us/1.5.0.md:47]: change what?

**Review by pengten — APPROVED** [review]

**Review by a364176773 — APPROVED** [review]
LGTM

**Review by slievrly — COMMENTED** [review]
LGTM

**Pull request merged**